docs-api/frontity-packages/features-packages/wp-source.md

```diff
@@ -110,6 +110,10 @@ export default {
 };
 ```
 
+{% hint style="info" %}
+If you are using [Embedded Mode](https://docs.frontity.org/architecture/embedded-mode) for your Frontity project, you do not normally have to set the `state.source.url` property as it will be the same as the `state.frontity.url` property.
```

Comment:
As we don't have an official place saying in the docs saying that `state.frontity.url` should be set in Frontity project and which value should it contain, I would rephrase this with something such as...

> If you are using [Embedded Mode](https://docs.frontity.org/architecture/embedded-mode) for your Frontity project and your `state.frontity.url` is set, you do not have to set the `state.source.url` property as it will be the same as the `state.frontity.url` property.